cognitedata/reveal
# fix: Incorrect behaviour of zoomToCursor mode after DefaultCameraManager deactivation/activation.

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-708

## Description :pencil:

Fixes a bug where `zoomToCursor` mode of `DefaultCameraManager` was behaving incorrectly after camera manager deactivation and then activation.

## How has this been tested? :mag:


In Reveal example with CAD and point cloud models, also on the phone with primitives model.

## Test instructions :information_source:



1. Open any model in Reveal example
2. Zoom to different points within the model and observe that camera is responsive and moving exactly towards position under the cursor
3. Change camera manager to `Custom` in `Camera controls` tab of menu GUI, rotate around a little bit and change it back to default.
4. Do the same as in step 2. and notice that behaviour is the same, before this PR it was incorrect as described in Jira issue.



## Checklist :ballot_box_with_check:
